assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -1,6 +1,5 @@
 import {Component, Fragment} from '@wordpress/element';
 import {ArrowIcon} from './ArrowIcon';
-import {toDeclarations} from '../toDeclarations';
```

Comment:
Good catch! We probably can even remove that file?

Reply:
Sure

Reply:
Can you please delete this file then?